src/Neo/Extensions/SmartContract/GasTokenExtensions.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// GasTokenExtensions.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Neo.Extensions
+{
+    public static class GasTokenExtensions
+    {
+        public static IEnumerable<(UInt160 Address, BigInteger Balance)> GetAccounts(this GasToken gasToken, DataCache snapshot)
```

Comment:
And BTW, why not to generalize this extension for `FungibleToken` instead of having two almost identical extensions defined on both Neo and Gas?

Reply:
Let's check it, in general it would be nice to unify these extensions.

Reply:
It won't work.

![image](https://github.com/user-attachments/assets/01b8e8fb-d5c8-4d73-a9a0-e20b88aab18c)